src/index.html

```diff
@@ -4,9 +4,360 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet"
+    crossorigin="true">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+
+      <header class="header">
+        <a href="#" class="navbar-logo">
+          <img
+          class="logo_img"
+          src="images/logo.png"
+          alt="logo"
+          >
```

Comment:
```suggestion
          <img
            class="logo_img"
            src="images/logo.png"
            alt="logo"
          >
```
Fix indents according to the code style guide